recipes/mlpy/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "3.5.0" %}
+{% set sha256 = "344fa75fbf9f76af72f6a346d5309613defc4d244bac13c218e509a51d68bf6a" %}
+#source file is named download, has to be unpacked manually, update build sh accordingly when updating version
+
+package:
+  name: mlpy
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/mlpy/files/mlpy%20{{ version }}/mlpy-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [osx]
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python >=2.6
+    - setuptools
+    - numpy =1.5
+    - scipy =0.11
+    - gsl =1.16
+  run:
+    - python >=2.6
+    - numpy =1.5
+    - scipy =0.11
+    - gsl =1.16
+
+
+test:
+  imports:
+    - mlpy
+
+about:
+  home: http://mlpy.sourceforge.net/index.html
+  license: GPL3
+  license_file: LICENSE.txt
```

Comment:
Not the right file. Or not the only one required. :-/ It says
```
mlpy: See gpl-3.0.txt
libsvm: See mlpy/libsvm/libsvm/COPYRIGHT
liblinear: See mlpy/liblinear/liblinear/COPYRIGHT
fastcluster: See gpl-3.0.txt
```

That's a bit of a mess. The ideal option would be to unbundle those and use the packages. At least fastcluster and libsvm have packages on conda-forge. I'm not sure about liblinear.

Alternatively, you'd probably have to compose a license file manually and add it to the recipe.

Reply:
License is fixed I hope, compiled a collection, they do not ship the COPYRIGHT files as they claim, but this is an upstream problem IMHO, also version ranges seem to work, thanks again for that neat trick